_i18n/en.yml

```diff
@@ -1149,24 +1149,24 @@ index:
   why:
     heading: |-
       ## Login.gov is for you
-    developers: |-
-      <h4 class="hdr developers">Agency developers</h4>
-
-      Developer resources, real-time support and modern tools to help you implement and deploy your application with login.gov
-
-      [See developer guide](https://developers.login.gov){:class="why-more-info" :target="_blank"}
     individuals: |-
-      <h4 class="hdr individuals">Individuals</h4>
+      <h3 class="hdr individuals">Individuals</h3>
 
       Use one account for secure, private access to participating government agencies.
 
       [Learn about login.gov](site.baseurl/what-is-login/){:class="why-more-info"}
     partners: |-
-      <h4 class="hdr partners">Agency partners</h4>
+      <h3 class="hdr partners">Agency partners</h3>
 
       Protect your users' information with the highest standards of digital security and user experience.  Login.gov handles software development, security operations, and customer support so you don't have to.
 
       [Become a partner](https://partners.login.gov/){:class="why-more-info" :target="_blank"}
+    developers: |-
+      <h3 class="hdr developers">Agency developers</h3>
+
+      Developer resources, real-time support and modern tools to help you implement and deploy your application with login.gov
+
+      [See developer guide](https://developers.login.gov){:class="why-more-info" :target="_blank"}
```

Comment:
Not for here, but considering these changes regard accessibility, I feel compelled to note:

1. The `target` doesn't actually seem to be assigned to the links
2. If it did, I imagine there would be concern for [G201: Giving users advanced warning when opening a new window](https://www.w3.org/TR/WCAG20-TECHS/G201.html)